src/main/java/uk/nhs/digital/nhsconnect/nhais/model/sequence/SequenceId.java

```diff
@@ -0,0 +1,15 @@
+package uk.nhs.digital.nhsconnect.nhais.model.sequence;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@AllArgsConstructor
+@Document(collection = "sequence")
```

Comment:
Maybe something like "send_sequence" or "outbound_sequence"? There is also a sequence concept for inbound.

I missed this for the outbound state - collection name is just the java class name!

Reply:
renaming class to OutboundSequenceId